config/site.navbar.config.js

```diff
@@ -1,3 +1,7 @@
+require("dotenv").config({
+    path: process.env.NODE_ENV ? `.env.${process.env.NODE_ENV}` : `.env`,
+});
+
```

Comment:
```suggestion
```

This is unnecessary, as `dotenv` is already called in parent config `docusaurus.config.js`.

Reply:
You're right, but this may be an issue when developing that in some cases the env file may not be accessed. This is a fail-safe for devs, but shouldn't be an issue on production

Reply:
Having such code duplicated makes code much harder to maintain. If you really need this fail-safe, please come up with utility that can be called from multiple places.

Reply:
I accepted your suggestion for now. We're working on other changes so I will look for a better approach for this for the next PRs